main.js

```diff
@@ -148,7 +148,7 @@ module.exports = function(options) {
 				directory + '/views/partials'
 			],
 			defaultLayout: false,
-			layoutsDir: options.layoutsDir || __dirname + '/layouts',
+			layoutsDir: options.layoutsDir || process.cwd() + '/client/layouts',
```

Comment:
But then you force every app to copy the layouts across to client, or manually set the layouts directory. I think the default should definitely be to point to the bower component. `nbt verify` can be tweaked to break builds if n-layout is not installed